src/engine.cpp

```diff
@@ -256,7 +256,7 @@ void LuaLogError(const char* msg) { if (msg) { LOG_NOFUNC_ERROR(msg); } else { L
 
 void Engine::InitSubSystems()
 {
-    mRenderer = std::make_unique<Renderer>((mFileSystem->FsPath() + "data/Roboto-Regular.ttf").c_str());
+    mRenderer = std::make_unique<Renderer>((mFileSystem->FsPath() + "data/Roboto-Regular.ttf").c_str(), (mFileSystem->FsPath() + "data/Roboto-Italic.ttf").c_str());
```

Comment:
you need to commit this font file, its not in the repo. Also add to cmakelists.txt with the other font otherwise it won't get installed